Automattic/jetpack
# Admin Page: Add confirm message when changing routes, informing about unsaved options

Fixes #4982 .

#### Changes proposed in this Pull Request:

Adds a mechanism for warning the user that some module settings have not been saved when trying to change tabs or routes.

#### Testing instructions:

1. Get to the **Site Stats** under the  **Engagement** tab 
1. update one of the options, without saving.
1. Switch to the **Security** tab
1. Expect to be prompted
